### DISCUSSION TITLE
Organic Resources

### DISCUSSION THREAD

**Discussion by DeHades666** [aquasarus/Starfield-Interactive-Smart-Slate#7] [Ideas]
Apologies for another topic, and another inept attempt at describing the idea.. Kudos for the slate though. 👍 
The Organic Resource Search tab is probably the most invaluable for me =) 
Okay, down to business. Let's say I need some Lubricant.. again just an example, I see that the resource is on Volii Epsilon and obtained from Flocking Trident Grazers ( just an example from the data I have in my slate). I ask myself, can I set up an outpost and farm these? At the moment I click back to planetary data find the planet, check the fauna and read the note that says, well it says "Prod allowed", because that's what I typed in from my bit of paper. Would it be possible to have a tickbox or something similar on the lifeform overview to say if they can be domesticated or stuck in the barns or whatever the term is, and then show up in the search results next to the creature to say it can be farmed/domesticated whatever ? Maybe a click through from "Resources Found in.." or "...on these Faunas" if not to take you back to the planetary data or lifefoem overview?
Apologies again, for the idea/request and the convoluted way about it, but if it makes people feel better and less annoyed, it took me almost an hour to write this.. words for the "words" don't come easily to me thesedays, having a conversation is like having a really bad game of charades with many many pauses =s

**Comment by aquasarus** [comment]
Okay this is actually pretty helpful. I wasn't sure how to go about the additional lifeform attributes because in my 200 hours of gameplay, I never ended up farming any fauna/flora. Your example makes it pretty clear that a "can be domesticated/farmed" checkbox might be the best way to go. I'm gonna do a bit more research to make sure we have all the edge cases covered, but yes this definitely seems like important data to integrate into a smart slate ;).

And don't worry about thinking out loud! This place is for brainstorming, and I am happy to comb through any word-vomits you'd like to lay out.
